package.json

```diff
@@ -32,22 +32,22 @@
   "devDependencies": {
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@tsconfig/node16": "^1.0.3",
-    "@types/node": "^16.11.56",
-    "@types/vscode": "^1.78.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
-    "@typescript-eslint/parser": "^5.59.2",
-    "@vscode/extension-telemetry": "^0.6.2",
-    "@vscode/vsce": "^2.19.0",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "eslint": "^8.40.0",
-    "eslint-config-universe": "^11.2.0",
-    "nodemon": "^2.0.22",
-    "ovsx": "^0.8.1",
-    "prettier": "^2.8.8",
-    "semantic-release": "^19.0.3",
+    "@tsconfig/node16": "^16.1.1",
```

Comment:
Might be worth to bump node to 18 as well, since 16 is EOL 😄

Reply:
Yeah, can look at that. 👍  

Also the latest version of semantic release package requires Node 18, so bumping Node will allow us to updated that package too.

<img width="972" alt="Screenshot 2023-10-07 at 13 29 19" src="https://github.com/expo/vscode-expo-theme/assets/719641/41a17251-a43a-4ca8-95a6-61572ea4b515">